R/ipsum.r

```diff
@@ -90,7 +91,6 @@ ipsum <- function(number_sections = FALSE,
     clean_supporting = self_contained,
     pre_processor = pre_processor,
     base_format = rmarkdown::html_document_base(
-      smart = smart,
```

Comment:
This is the one code change. Since this is getting passed through to `...` and ignored, it should not affect behavior.